rust/syntect-plugin/src/main.rs

```diff
@@ -249,15 +249,24 @@ impl<'a> Syntect<'a> {
                 //TODO: view should have a `get_edit_builder` fn?
                 let mut builder = EditBuilder::new(buf_size);
                 builder.replace(interval, indent.into());
+
                 let delta = builder.build();
-                return Some(PluginEdit {
-                    rev: rev,
-                    delta: delta,
+                Some(delta)
+            } else {
+                None
```

Comment:
could we not just return in the case that there's no delta? Then `result` doesn't need to be an option, and the following code is a bit cleaner.

Reply:
I could not call the edit function on the view while the view is mutably borrowed for "get_line", thus I used 'if expression' to return the result instead and utilize it later. The same thing I discussed in IRC.